pkg/instance/instance.go

```diff
@@ -134,8 +135,9 @@ func (i *Instance) Included() []jsonapi.Object {
 
 // settings is a struct used for the settings of an instance
 type instanceSettings struct {
-	Timezone string `json:"tz,omitempty"`
-	Email    string `json:"email,omitempty"`
+	Timezone   string `json:"tz,omitempty"`
+	Email      string `json:"email,omitempty"`
+	PublicName string `json:"public_name,omitempty"`
```

Comment:
Would be interesting to add the possibility to specify this public name in the:
  - `/instance` API (defined in `web/instance` via a `?PublicName` query param)
  - in the client (`client/instances#CreateInstance` via a `PublicName` field in the `InstanceOptions` struct)
  - in the CLI (`cmd/instances#addInstanceCmd`, via a `--public-name` flag for instance)